.github/workflows/build.yml

```diff
@@ -1,9 +1,15 @@
 name: OpenDCS Build and unit tests
 on:
   pull_request:
-    branches: ["master"]
+    branches:
+      - "main"
+      - "7.0"
+      - "8.0"
```

Comment:
If this is the 7.0 branch, should we remove main and 8.0 PR's?   same question for push.

Reply:
Honestly, I'm not entirely sure. We'll need to do a little research, but for know I at least know it's not breaking anything to do it this way.